docs/op-stack/src/docs/build/sdk.md

```diff
@@ -1,17 +1,24 @@
 ---
-title: Using the SDK with OP Stack
+title: Using the OP Stack Client SDK
 lang: en-US
 ---
 
-When building applications for use with your OP Stack, you can continue to use [the Optimism JavaScript SDK](https://sdk.optimism.io/).
-The main difference is you need to provide some contract addresses to the `CrossDomainMessenger` because they aren't preconfigured.
 
+## Natively supported chains
 
-## Contract addresses
+[The OP Stack Client SDK](https://sdk.optimism.io/) natively supports multiple OP Chains: OP, Base, Zora, etc.
+To see whether a specific OP Chain is supported directly, [see the documentation](https://sdk.optimism.io/enums/l2chainid).
```

Comment:
Same comment from:
- https://github.com/ethereum-optimism/optimism-tutorial/pull/194
- https://github.com/ethereum-optimism/community-hub/pull/831
Zora and Base aren't mentioned in the linked sdk documentation